simplecloud-modules/simplecloud-module-placeholders/src/main/kotlin/eu/thesimplecloud/module/placeholders/spigot/BukkitPluginMain.kt

```diff
@@ -0,0 +1,46 @@
+package eu.thesimplecloud.module.placeholders.spigot
+
+import eu.thesimplecloud.module.placeholders.spigot.placeholderapi.PlaceholderHook
+import org.bukkit.plugin.java.JavaPlugin
+
+class BukkitPluginMain : JavaPlugin() {
+
+    var replacePermissionModule = false
+    var replaceChatTabModule = false
+
+    override fun onEnable() {
+
+        checkPermissionsModule()
+        checkChatTabModule()
+
+        val register = PlaceholderHook(this).register()
+
+        if(register) println("[INFO] Registered PlaceholderAPI hook successfully!")
+        else println("[FATAL] Failed to register PlaceholderAPI hook!")
+    }
+
+    fun checkPermissionsModule() {
+        try {
+            val forName = Class.forName("eu.thesimplecloud.module.permission.manager.PermissionModule")
```

Comment:
Please just use Bukkit to look, if the plugin exists

Reply:
Now it's being done by checking if the plugin is null with checking if Bukkit#getPluginManager#getPlugin isn't null